scripts/vpn-port-guard.sh

```
@@ -1,361 +1,292 @@
+  if qbt_post "/api/v2/torrents/resume" --data "hashes=all"; then
+    _qbt_state="active"
+    return 0
+  fi
+  _qbt_state="error"
+  return 1
+}
 
-  controller_pause_qbt || true
-  qbt_api_login || controller_log "Unable to authenticate with qBittorrent API yet"
+initialise() {
+  write_status "init" 0 "$CONTROLLER_REQUIRE_PF" "unknown" "initializing" || true
```

Shellcheck (0.11.0)

[warning] Did you forget to close this double quoted string?

(SC1078)